src/server/index.ts

```diff
@@ -66,6 +74,7 @@ function startServer() {
 
   const clientPath = getClientPath();
 
+  setupCookieParser(app);
```

Comment:
This will set up the middleware to sign our game host cookie. The cookie is also going to be HttpOnly, but this should also hopefully help safeguard us a bit against stuff like cookie jar overflow, to verify that the cookie has not been tampered with.